### PR TITLE
[typo] comment & long code line & simplify lambda

### PR DESCRIPTION
**Pull request by jxd134**
some minor modify

### CODE REVIEW

File: common/src/main/java/com/megaease/easeagent/common/HostAddress.java [hunk 1]
```diff
@@ -96,17 +96,15 @@ public static String initLocalHostname() {
...
-                        final NetworkInterface nic = interfaces.nextElement();
-                        final Enumeration<InetAddress> addresses = nic.getInetAddresses();
-                        while (addresses.hasMoreElements()) {
-                            final InetAddress address = addresses.nextElement();
-                            if (!address.isLoopbackAddress()) {
-                                final String hostname = address.getHostName();
-                                if (hostname != null) {
-                                    return hostname;
-                                }
+                while (interfaces.hasMoreElements()) {
```
Comment (akwei) [inline common/src/main/java/com/megaease/easeagent/common/HostAddress.java:99]: interfaces maybe are null in the document. We should check it

Comment (jxd134) [reply common/src/main/java/com/megaease/easeagent/common/HostAddress.java:99]: @akwei yes, it maybe null, but if the interfaces is null, the method will throw a `SocketException`.
图片

Comment (akwei) [reply common/src/main/java/com/megaease/easeagent/common/HostAddress.java:99]: In OpenJDK
image

Comment (jxd134) [reply common/src/main/java/com/megaease/easeagent/common/HostAddress.java:99]: My jdk is corretto-11.
You are right, we should check it for defensive programming。

**Review by akwei — APPROVED** [review]

**Pull request merged**